packages/test/package.json

```diff
@@ -46,7 +46,7 @@
     "url": "https://github.com/coinbase/rest-hooks/issues"
   },
   "dependencies": {
-    "@testing-library/react-hooks": "~3.5.0"
+    "@testing-library/react-hooks": "~5.0.3"
```

Comment:
why ~ ?

Reply:
As outlined in a previous PR - there was an uncaught breaking change in a minor upgrade to this library. This lets us validate all minor upgrades before our users potentially get broken things.

Reply:
oof, gross